App/src/libreria/authenticationGoogle.js

```diff
@@ -0,0 +1,59 @@
+import {
+  GoogleSignin,
+  statusCodes,
+} from '@react-native-google-signin/google-signin';
+import auth from '@react-native-firebase/auth';
+import firestore from '@react-native-firebase/firestore';
+//const signIn = (firstname, suscribe, email, password, navigation)
+export const signInGoogle = async navigation => {
+  try {
+    GoogleSignin.configure({
+      webClientId:
+        '1023121760646-4o0p6e0meeogfe9he5poq2llq4oo873e.apps.googleusercontent.com',
```

Comment:
Your Google ID should not appear on GitHub, use environment variables to hide it.